pkg/resources/task.go

```diff
@@ -619,6 +635,21 @@ func UpdateTask(d *schema.ResourceData, meta interface{}) error {
 		}
 	}
 
+	if d.HasChange(AllowOverlappingExecution) {
+		var q string
+		n := d.Get(AllowOverlappingExecution)
+		flag := n.(bool)
+		if flag == false {
+			q = builder.UnsetAllowOverlappingExecution()
+		} else {
+			q = builder.ChangeAllowOverlappingExecution(flag)
```

Comment:
since this function only gets called if flag is true, why not simplify to:
```
if !flag {
 q = builder.UnsetAllowOverlappingExecution()
} else {
q = builder.SetAllowOverlappingExecution()
}
```